react/features/base/toolbox/components/ToolboxItem.web.js

```diff
@@ -21,23 +21,40 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
      */
     _renderItem() {
         const {
+            disabled,
             onClick,
-            showLabel
+            showLabel,
+            tooltipPosition
         } = this.props;
+        const className = showLabel ? 'overflow-menu-item' : 'toolbox-button';
         const props = {
             'aria-label': this.accessibilityLabel,
-            className: showLabel ? 'overflow-menu-item' : 'toolbox-button',
-            onClick
+            className: className + (disabled ? ' disabled' : ''),
+            onClick: disabled ? undefined : onClick
         };
         const elementType = showLabel ? 'li' : 'div';
+        const useTooltip = this.tooltip && this.tooltip.length > 0;
         // eslint-disable-next-line no-extra-parens
-        const children = (
+        let children = (
```

Comment:
I think the logic can be simplified here. If a tooltip is used you don't need this `Fragment` AFAICT. You can just use the `Tooltip` as the parent.

Reply:
No, ```Tooltip``` can only have one child

Reply:
Doh!